packages/contracts-bedrock/test/L2/SuperchainWETH.t.sol

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
+
+// Contract imports
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { IL2ToL2CrossDomainMessenger } from "src/L2/IL2ToL2CrossDomainMessenger.sol";
+import { ETHLiquidity } from "src/L2/ETHLiquidity.sol";
+
+// Error imports
+import "src/libraries/errors/CommonErrors.sol";
+
+/// @title SuperchainWETH_Test
+/// @notice Contract for testing the SuperchainWETH contract.
+contract SuperchainWETH_Test is CommonTest {
+    /// @notice Emitted when a transfer is made.
+    event Transfer(address indexed src, address indexed dst, uint256 wad);
```

Comment:
Just fyi, on modern versions of solc, events are available as properties on a contract, so you don't need to define them twice

Reply:
Not on `0.8.15` tho

Reply:
Will make this change when we update to a more recent solc.